src/main/java/com/cupid/jikting/member/service/FileUploadService.java

```diff
@@ -66,14 +74,19 @@ private void validateExtension(String extension) {
     }
 
     private void validateFace(String fileName) {
+        AWSCredentials awsCredentials = new BasicAWSCredentials(accessKey, secretKey);
+
         DetectFacesRequest request = new DetectFacesRequest()
                 .withImage(new Image()
                         .withS3Object(new S3Object()
                                 .withName(fileName)
                                 .withBucket(bucket)))
                 .withAttributes(Attribute.DEFAULT);
         try {
-            AmazonRekognitionClientBuilder.defaultClient()
+            AmazonRekognitionClientBuilder.standard()
+                    .withCredentials(new AWSStaticCredentialsProvider(awsCredentials))
```

Comment:
p3. awsCredentials는 바로 사용하는건 어떨까요?

```suggestion
        DetectFacesRequest request = new DetectFacesRequest()
                .withImage(new Image()
                        .withS3Object(new S3Object()
                                .withName(fileName)
                                .withBucket(bucket)))
                .withAttributes(Attribute.DEFAULT);
        try {
            AmazonRekognitionClientBuilder.defaultClient()
            AmazonRekognitionClientBuilder.standard()
                    .withCredentials(new AWSStaticCredentialsProvider(new BasicAWSCredentials(accessKey, secretKey)))
```

Reply:
수정완료